src/lib/utils/synapseTypes/Table/TableUpdate.ts

```diff
@@ -0,0 +1,36 @@
+// http://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/table/TableUpdateTransactionRequest.html
+// Update a table
+export type TableUpdateTransactionRequest = {
+  concreteType: 'org.sagebionetworks.repo.model.table.TableUpdateTransactionRequest'
+  entityId: string
+  changes: AppendableRowSetRequest[] //List of changes that describes schema and/or row changes to a table.
+  // createSnapshot?: boolean
```

Comment:
would having the 'createSnapshot' options set serve any sort of record keeping utility?

Reply:
Yes, it's a way to automatically create a version of the Table/View.  If we add this functionality, I'd rather make this an explicit call (rather than combining with a Table/View Update).